### DISCUSSION TITLE
How do I connect Wildfly26 to a SAML IDP?

### DISCUSSION THREAD

**Discussion by grldchz** [keycloak/keycloak#16600] [Q&A]
I have used the Keycloak SAML adapter for years.  I recently attempted to upgrade to Wildfly 26 and found that Keycloak only supports a SAML adapter for Wildfly <= 23.  

Is there a suggested solution for connecting WIldfly 26 to a SAML IDP?

**Comment by stianst** [comment]
We are working on supporting more recent versions of WildFly with the SAML adapter. That should be ready in Keycloak 22.

@pskopek FIY

**Comment by grldchz** [comment]
No problem.  I was able to get it working with Keycloak 21 by creating my
own custom wildfly galleon layer.

Thanks,
Nate
